src/univariate/continuous/exponential.jl

```diff
@@ -67,7 +71,8 @@ xval(d::Exponential, z::Real) = z * d.θ
 
 pdf(d::Exponential, x::Real) = (λ = rate(d); x < 0 ? zero(λ) : λ * exp(-λ * x))
 function logpdf(d::Exponential{T}, x::Real) where T<:Real
-    (λ = rate(d); x < 0 ? -T(Inf) : log(λ) - λ * x)
+    λ = rate(d)
+    x < 0 ? -T(Inf) : log(λ) - λ * x
```

Comment:
This will fail if `T<:Integer`

Reply:
Which cannot happen by construction:
```julia
Exponential(θ::Integer) = Exponential(Float64(θ))
```

Reply:
OK - I see that we have a conversion to float in the constructor